packages/mdc-textfield/README.md

```diff
@@ -176,24 +185,55 @@ UX for client-side form field validation.
 </p>
 ```
 
-### Multi-line - With Javascript
+### Leading and Trailing Icons
+Leading and trailing icons can be added to MDC Textfields as visual indicators
+as well as interaction targets. To do so, add the relevant classnames to the root element, add
+an `i` element with your preferred icon, and give it either a `mdc-textfield__leading-icon` or
+`mdc-textfield__trailing-icon` class depending on which you are looking to achieve.
+
+#### Leading:
+```html
+<div class="mdc-textfield mdc-textfield--box mdc-textfield--with-leading-icon">
+  <i class="material-icons mdc-textfield__leading-icon" tabindex="0">event</i>
+  <input type="text" id="my-input" class="mdc-textfield__input">
+  <label for="my-input" class="mdc-textfield__label">Your Name</label>
+  <div class="mdc-textfield__bottom-line"></div>
+</div>
+```
+
+#### Trailing:
+```html
+<div class="mdc-textfield mdc-textfield--box mdc-textfield--with-trailing-icon">
+  <i class="material-icons mdc-textfield__trailing-icon" tabindex="0">event</i>
```

Comment:
You show the trailing icon element before the input and label in this doc example, but in the actual demo, the trailing icon element is after these two elements. Does either work, or is this example incorrect?

Reply:
Either works, but I think in this case it makes sense to align with the demo. Makes for a better mental model.